.changeset/tame-bears-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`AbstractSigner`, `SignerECDSA`, `SignerP256`, and `SignerRSA`: Add an abstract contract and various implementations for contracts that deal with signature verification.
```

Comment:
AbstractSigner already added in #5664

Reply:
Oh yes I need to rebase this branch

Reply:
Oh but this one doesn't have a changeset, so I think it's cool adding it here